chainer/functions/connection/linear.py

```diff
@@ -57,10 +57,10 @@ def backward(self, indexes, grad_outputs):
 
         ret = []
         if 0 in indexes:
-            gx = linear(gy, W.T)
+            gx, = LinearGradData(self).apply((gy, W))
```

Comment:
I think it is better to align the argument order with `LinearGradWeight`.
(i.e., `apply((W, gy))` instead of `apply((gy, W))`)

Reply:
ok, I will modify it.